src/main/kotlin/vm/VM.kt

```diff
@@ -202,10 +204,24 @@ class VM(val exe: Executable) {
                         } ?: Yegg.world.sys.callVerb(c, name, args)?.also {
                             result = it
                         } ?: fail(E_VARNF, "no such fun or variable")
-                        result?.also { push(it) }
+                        result?.also { if (word.opcode == O_FUNCALL) push(it) }
                     }
                 }
 
+                // Task ops
+
+                O_SUSPEND -> {
+                    val a = pop()
+                    throw SuspendException((a as VInt).v)
```

Comment:
This is how I speculatively intend to implement suspend, but I haven't done that yet.